tello_driver:
  ros__parameters:
    # Topics
    image_topic_name: '/camera/image_raw'
    flight_data_topic_name: '/flight_data'
    velocity_command_topic_name: '/cmd_vel'
    land_topic_name: '/land'
    takeoff_topic_name: '/takeoff'
    flip_control_topic_name: '/flip'

    auto_wifi_connection: False
    tello_ssid: 'tello01'
    tello_pw: ''
